src/SideBar/SettingsPane/styles.module.scss

```diff
@@ -18,6 +18,7 @@
 
 .tab {
   padding: 1rem 1.5rem;
+  width: var(--sidebar-width);
```

Comment:
The settings panel was redimensioning its width in each tab. This keeps it static.

Reply:
Any downsides here? overflow/trimming effects as a result of this?

Reply:
The issue we had is that it was increasing its width dynamically with its content. Long texts increased its width until it covered all the screen. I didn't see anything wrongly affected.